src/docs/santiment-queries/liquidation/index.md

```diff
@@ -0,0 +1,95 @@
+---
+title: Liquidation Overview
```

Comment:
I don't think that Liquidations is the right name. In the article I see that we track not only liquidations but also other lending pool events. I would suggest to use *Lending Pools*

Reply:
This change should also be applied to several other places where the word *Liquidations* is used